### ISSUE TITLE
Question: What to do with unsure.sql

### ISSUE THREAD

**Issue by jfchen48** [dalibo/sqlserver2pgsql#104]
Just a question: in the script to migrate the data, the example runs the before.sql file and after.sql file. If we believe we have corrected the issues in unsure.sql, should the scripts/statements in this script be placed in before.sql or after.sql? Or does the file contain a mix of statements that we supposed to decide goes in either before or after? Thank you!

**Comment by madtibo** [comment]
Hello jfchen48,

The commands in unsure.sql are the trickiest one where the parser can make errors. They must be verified to make sure it is in tune with your original commands.
Once that is done, they can stay in they own unsure.sql file and be loaded after the after.sql file.
Tell me if loading them last cause some problems.

Cordialement,

**Comment by manishdwivedi02** [comment]
Just wanted to check, are you writing queries obtained from SQL DB as is in unsure.sql, and I have to validate each and every query before running this script.  For example there are in-built function in SQL DB, which may by different in PostgreSQL DB and I see many other issue in unsure.sql. Few are given below

1) CHECK (datalength(fieldname)>(0)) – Getting error, since datalength function does not exist in PostgreSQL
2) b1.C1 +  '|'  + b2.C1 – This is generated in create view query. However this should be concat(concat(b1.c1, '|'), b2.c1) AS c1 in PostgreSQL. 
3) convert(NVARCHAR(15),CO12345) + ';' + convert(NVARCHAR(15),CC12345) + ';' + C12345 – This is from create view query and this directly does not work on PostgreSQL. I think I have to use concat function, but problem is there are many occurrence of this kind

**Comment by madtibo** [comment]
@manishdwivedi02 if you encounter some problems, please open a new issue and give the original SQL server dump so we can investigate your problem.

**Comment by madtibo** [comment]
@jfchen48 I close this issue, do not hesitate to tell me if your question was not entirely answered
